core/templates/filters/string-utility-filters/sort-by.pipe.ts

```diff
@@ -35,10 +35,6 @@ export class SortByPipe implements PipeTransform {
   transform<T>(
       value: T[], isDescending: boolean,
       sortKey?: string): T[] {
-    if (!value) {
```

Comment:
The generic type `T` does not allow `null` or `undefined` values as an argument.